src/crimson/os/seastore/lba_manager/btree/btree_lba_manager.cc

```diff
@@ -373,55 +373,78 @@ BtreeLBAManager::scan_mapped_space_ret BtreeLBAManager::scan_mapped_space(
     });
 }
 
-BtreeLBAManager::rewrite_extent_ret BtreeLBAManager::rewrite_extent(
+BtreeLBAManager::rewrite_extent_ret BtreeLBAManager::rewrite_logical_extent_p1(
```

Comment:
This method doesn't actually seem to need to be in btree_lba_manager at all.  This logic was here before I think because btree_lba_manager simply handled the rewrite in its entirety, but now that you are distinguishing between logical and physical extents at a higher level, you can move this logic up into the caller.

Reply:
I think this method also has the only epm reference in btree_lba_manager, so you could remove that as well.